pom.xml

```diff
@@ -665,3920 +650,530 @@
-                    </exclusion>
-                    <exclusion>
-                        <groupId>xml-apis</groupId>
-                        <artifactId>xml-apis</artifactId>
-                    </exclusion>
-                </exclusions>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-all</artifactId>
+                <version>${version.groovy-all}</version>
+                <scope>test</scope>
             </dependency>
-
+            <!-- Required by Wiremock for the MicroProfile REST Client TCK -->
             <dependency>
-                <groupId>${ee.maven.groupId}</groupId>
-                <artifactId>wildfly-testsuite-shared</artifactId>
-                <version>${ee.maven.version}</version>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-http</artifactId>
+                <version>${version.org.eclipse.jetty}</version>
                 <scope>test</scope>
             </dependency>
 
+            <!-- Required by Wiremock for the MicroProfile REST Client TCK -->
             <dependency>
-                <groupId>org.apache.directory.server</groupId>
-                <artifactId>apacheds-test-framework</artifactId>
-                <version>${version.org.apache.ds}</version>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-io</artifactId>
+                <version>${version.org.eclipse.jetty}</version>
                 <scope>test</scope>
             </dependency>
-
+            <!-- Required by Wiremock for the MicroProfile REST Client TCK -->
             <dependency>
-                <groupId>org.apache.directory.server</groupId>
-                <artifactId>apacheds-interceptor-kerberos</artifactId>
-                <version>${version.org.apache.ds}</version>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-server</artifactId>
+                <version>${version.org.eclipse.jetty}</version>
                 <scope>test</scope>
             </dependency>
-
+            <!-- Required by Wiremock for the MicroProfile REST Client TCK -->
             <dependency>
-                <groupId>org.apache.openjpa</groupId>
-                <artifactId>openjpa-all</artifactId>
-                <version>${version.org.apache.openjpa}</version>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-util</artifactId>
+                <version>${version.org.eclipse.jetty}</version>
                 <scope>test</scope>
             </dependency>
 
             <dependency>
-                <groupId>org.awaitility</groupId>
-                <artifactId>awaitility</artifactId>
-                <version>${version.org.awaitility.awaitility}</version>
-                <scope>test</scope>
+                <groupId>org.hamcrest</groupId>
+                <artifactId>hamcrest-all</artifactId>
+                <version>${version.org.hamcrest}</version>
```

Comment:
Missing test scope?... there are some of them with the same missing scope

Reply:
Thanks I'll add it to this one.

Some of the others are used for compiling non src/test/java code. Some of the ARQ-related ones, I'm not sure if they are or not but in at least 1 case I added the test scope and the build broke, so I stopped doing that. I'm sure it can be added to some though, if someone has time to determine which.

Note that testsuite/shared has its code in src/main/java so any seemingly 'test' dep it has can't have test scope. That was the ARQ problem I hit.

Reply:
> Note that testsuite/shared has its code in src/main/java 

ahh, ok, I missed that detail

Reply:
I fixed the hamcrest-all one.